### PR TITLE
Add link_traits recipe

### PR DESCRIPTION
**Pull request by francisco-dlp**

### CODE REVIEW

File: recipes/link_traits/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package:
```
Comment (synapticarbors) [inline recipes/link_traits/meta.yaml]: Please use jinja template variables as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L6

File: recipes/link_traits/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  name: link-traits
+  version: "1.0.1"
+
+source:
+  fn: link_traits-1.0.1.tar.gz
+  url: https://pypi.python.org/packages/64/d1/4d683994fbba2d3691f7433c09453ea8f4a5144a74767bd39c0d0f8ad25a/link_traits-1.0.1.tar.gz
```
Comment (synapticarbors) [inline recipes/link_traits/meta.yaml]: Please use the canonical pypi.io link as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

File: recipes/link_traits/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  md5: 7f270bc64ccd606573e9dd1d929f0f63
```
Comment (synapticarbors) [inline recipes/link_traits/meta.yaml]: please use `sha256` instead of `md5`.

File: recipes/link_traits/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - traits
```
Comment (synapticarbors) [inline recipes/link_traits/meta.yaml]: `traits` shouldn't be a `build` requirement.

File: recipes/link_traits/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    - traitlets
+
+about:
+  home: https://github.com/hyperspy/link_traits
+  license: BSD
+  summary: "A fork to traitlets' link and dlink to link traits in addition to traitlets."
+  license_family: BSD
+extra:
+  recipe-maintainers:
+    - to266
```
Comment (synapticarbors) [inline recipes/link_traits/meta.yaml]: @to266 are you ok being added as a co-maintainer of the `link_traits` package on conda-forge?

File: recipes/link_traits/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  commands:
+    - py.test --pyargs link_traits
+
+  requires:
+    - pytest
+    - traitlets
+
+about:
+  home: https://github.com/hyperspy/link_traits
+  license: BSD
```
Comment (synapticarbors) [inline recipes/link_traits/meta.yaml]: `BSD-3-Clause` instead of just `BSD`.

File: recipes/link_traits/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  summary: "A fork to traitlets' link and dlink to link traits in addition to traitlets."
+  license_family: BSD
```
Comment (synapticarbors) [inline recipes/link_traits/meta.yaml]: Please add the license to the `MANIFEST.in` in the upstream source repo and temporarily package the license with the recipe as in:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

File: recipes/link_traits/meta.yaml (hunk 5 above)
Comment (to266) [inline recipes/link_traits/meta.yaml]: I am

**Comment by francisco-dlp** [comment]
@synapticarbors, thanks for the quick review. I have fixed the recipe following your suggestions. However, I didn't manage to make the "canonical pypi link" work. Any suggestions?

**Comment by synapticarbors** [comment]
It looks like the file name uses an underscore rather than a dash:

https://pypi.org/project/link-traits/#files

**Comment by francisco-dlp** [comment]
Thanks!

**Comment by francisco-dlp** [comment]
It should be fixed now. Sorry that it took three attempts.

**Comment by francisco-dlp** [comment]
@synapticarbors, thanks for your help. Is there anything else that I should fix?

**Comment by francisco-dlp** [comment]
Thanks!

**Pull request merged**